src/patternfly/components/Menu/menu.scss

```diff
@@ -0,0 +1,252 @@
+.pf-c-menu {
+  // Menu
+  --pf-c-menu--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-menu--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-menu--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu--PaddingBottom: var(--pf-global--spacer--sm);
+
+  // Divider
+  --pf-c-menu--c-divider--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-menu--c-divider--MarginBottom: var(--pf-global--spacer--sm);
+
+  // Header
+  --pf-c-menu__header--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu__header--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__header--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__header--PaddingLeft: var(--pf-global--spacer--md);
+
+  // Menu item
+  --pf-c-menu__item--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-menu__item--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-menu__item--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-menu__item--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu__item--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-menu__item--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-menu__item--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-menu__item--disabled--Color: var(--pf-global--Color--dark-200);
+
+  // list item
+  --pf-c-menu__list-item--Color: var(--pf-global--Color--100);
+  --pf-c-menu__list-item--hover--Color: var(--pf-global--Color--100);
+  --pf-c-menu__list-item--BackgroundColor: transparent;
+  --pf-c-menu__list-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  // group
+  --pf-c-menu__group-title--PaddingTop: var(--pf-c-menu__item--PaddingTop);
+  --pf-c-menu__group-title--PaddingRight: var(--pf-c-menu__item--PaddingRight);
+  --pf-c-menu__group-title--PaddingBottom: var(--pf-c-menu__item--PaddingBottom);
+  --pf-c-menu__group-title--PaddingLeft: var(--pf-c-menu__item--PaddingLeft);
+  --pf-c-menu__group-title--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-menu__group-title--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-menu__group-title--Color: var(--pf-global--Color--dark-200);
+
+  // description
+  --pf-c-menu__item-description--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-menu__item-description--Color: var(--pf-global--Color--200);
+
+  // icon
+  --pf-c-menu__item-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-menu__item-icon--Color: var(--pf-global--Color--dark-200);
```

Comment:
Are we sure the icon shouldn't just match the current text color? It does in the current dropdown implementation - https://www.patternfly.org/v4/documentation/core/components/dropdown#split-button-action

Reply:
@mcarrano @mceledonia @doruskova what do you guys think ?

Reply:
This is the current dropdown:

<img width="340" alt="Screen Shot 2020-08-18 at 1 59 26 PM" src="https://user-images.githubusercontent.com/20118816/90549429-81d9d900-e15c-11ea-9008-465e5ded9828.png">

Reply:
**This is what i have in the menu:**
**default:**
<img width="467" alt="Screen Shot 2020-08-18 at 2 09 34 PM" src="https://user-images.githubusercontent.com/20118816/90549483-98803000-e15c-11ea-9699-0632083b63ff.png">
**hover/focus:**
<img width="471" alt="Screen Shot 2020-08-18 at 2 09 38 PM" src="https://user-images.githubusercontent.com/20118816/90549486-9918c680-e15c-11ea-82d2-aa31946fb8f3.png">

Reply:
@christiemolloy i think it makes sense to have it gray since it's a separate action and distinct from the menu item!

Reply:
Just to make sure we're referring to the same thing, this question is just about the icon on the left of the text. Here's an example of the icon when used with an item that also has an associated icon

what it looks like now
<img width="796" alt="Screen Shot 2020-08-18 at 1 37 33 PM" src="https://user-images.githubusercontent.com/35148959/90552222-1857cb80-e158-11ea-9aba-6fa53a1948a0.png">

what it would look like if the icon color matched the text color (like it does currently in the dropdown)
<img width="784" alt="Screen Shot 2020-08-18 at 1 37 44 PM" src="https://user-images.githubusercontent.com/35148959/90552220-1857cb80-e158-11ea-94eb-3d71fad477b0.png">

Reply:
oh my bad i was comparing the wrong icons 😩 , @gdoyle1 what do you think about cokers example , i think it makes sense for them to be the same as text since they dont have any hover/focus states

Reply:
@christiemolloy I agree with Michael, black makes more sense when the icon doesn't have any states.